package.json

```diff
@@ -1,6 +1,9 @@
 {
-  "name": "typescript-wallet-sdk",
+  "name": "@stellar/typescript-wallet-sdk",
```

Comment:
change package name to be consistent with other stellar libs

Reply:
should we update README instructions with `@stellar/typescript-wallet-sdk` as well?